### ISSUE TITLE
drop budgie-desktop overlay package

### ISSUE THREAD

**Issue by philmmanjaro** [manjaro/packages-community#149]
Since Archlinux adopted **budgie** we should think if we still need this package ...

**Comment by Ste74** [comment]
I think we can drop but whit v 10.2.6 version because is nearly to release
(ikey say for fixing fine 3.20 ). Now in our repo budgie is a pkgrel 2 but
not equal at arch tree because i have pushed 10.2.5 pkgrel 1 before and
yesterday pushed rel 2 equal rel 1 arch..
Il 10/apr/2016 12:32, "Philip Müller" <notifications@github.com> ha scritto:

Since Archlinux adopted *budgie* we should think if we still need this
package ...

—
You are receiving this because you were assigned.
Reply to this email directly or view it on GitHub
<https://github.com/manjaro/packages-community/issues/149>

**Comment by philmmanjaro** [comment]
Ok, makes sense as Arch has now also a revision 2 but with more patches ...

**Comment by Ste74** [comment]
Yes, i have to merge and push rel 3... in this moment i am not at home... i
pushed this evening.
Il 10/apr/2016 15:13, "Philip Müller" <notifications@github.com> ha scritto:

**Comment by philmmanjaro** [comment]
there is no rush at all in this ;)

**Comment by Ste74** [comment]
Lol ;) ... do you think for vertex maia theme and my idea to ask at Horst
to correct it?
Il 10/apr/2016 15:17, "Philip Müller" <notifications@github.com> ha scritto:
